keadm/cmd/keadm/app/cmd/util/reset.go

```diff
@@ -72,14 +73,17 @@ func CleanDirectories(isEdgeNode bool) error {
 	}
 
 	if isEdgeNode {
-		dirToClean = append(dirToClean, "/var/lib/dockershim", "/var/run/kubernetes", "/var/lib/cni")
```

Comment:
Don't we need to clean up these paths?

Reply:
I think keadm should not manage the path of container runtime, we can specify a cleanup script with `--post-run`. In addition, there is no /var/run/kubernetes path on the edge node.